kubernetes/create-services.sh.j2

```diff
@@ -8,7 +8,9 @@ kubectl create --record --filename services/verifier.yml
 {% endif %}
 kubectl create --record --filename services/geth.yml
 kubectl create --record --filename services/rabbitmq.yml
+{% if allow_signing_service_authentication %}
 kubectl create --record --filename services/middleman.yml
+{% endif %}
 {% if signing_service_on_the_cluster %}
```

Comment:
When not deploying middleman, you should also disable the TCP proxy in nginx.

Reply:
Done.